tests/server.py

```diff
@@ -280,6 +287,21 @@ def listen(self, factory: http.HTTPFactory) -> None:
 
 
 class WebSocketProtocol(WebSocketServerProtocol):
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        super().__init__(*args, **kwargs)
+        self.events = EventEmitter()
```

Comment:
this basically rewrites onMessage and onClose to an easier to consume pyee event emitter.